### ISSUE TITLE
[FEATURE] Audio Subtitles

### ISSUE THREAD

**Issue by xMGZx** [ApacheTech-VintageStory-Mods/ApacheTech.VintageMods.AccessibilityTweaks#9]
Audio subtitles are visual and textual cues for ingame audio type and direction. It's a super common accessibility option available in a variety of games, yet Vintage Story does not have them. For people who have trouble hearing in-game sounds for any specific reason would benefit heavily from subtitles.

A [subtitles mod](https://mods.vintagestory.at/show/mod/496) was once created for 1.14.8, but it is no longer updated anymore, and a more up-to-date mod would be desirable.

Is this feature within scope? I plan on making a subtitles mod either way, I was just curious if it would be practical to attempt to implement this feature into this mod or to keep it separate.
